File: youlexuan-sellergoods-interface/src/main/java/com/offcn/service/BrandService.java
package com.offcn.service;

import com.offcn.entity.PageResult;
import com.offcn.entity.Result;
import com.offcn.pojo.TbBrand;

public interface BrandService {

    /**
     * 分页查询
     * @param pageNum 第几页
     * @param pageSize 每页显示多少条
     * @return
     */
    PageResult selectBrand(int pageNum,int pageSize);

    /**
     * 新增品牌
     */
    Result add(TbBrand brand);

    /**
     *  删除品牌
     */
    Result delete(Long[] ids);


}

File: youlexuan-manager-web/src/main/webapp/js/service/brandService.js
//定义一个service层
app.service('brandService',function($http){
    this.delete = function(ids){
        return $http.get('../brand/deleteBrand.do?deleteIds='+ids);
    };
    this.add = function(entity){
        return $http.post('../brand/addBrand.do',entity);
    };
    this.findPage = function(page,rows){
        return $http.get('../brand/findPage.do?pageNum='+page+'&pageSize='+rows);
    }
});